### ISSUE TITLE
The email sent (with default config options) doesn't say which urls were problematic 

### ISSUE THREAD

**Issue by michaeljhopkins** [spatie/laravel-link-checker#1]
Rad package, and very easy to get set up and running. The email that was sent to me contained the following body

```
Found broken urls

Crawled 10 with status code 200
Crawled 1 with status code 999
Crawled 1 with status code 302
```

I'm not interested in the urls of the 200's, but it would be very helpful to get a little more info on the other status codes

**Comment by freekmurze** [comment]
Thanks! It was not my intention to mail non-broken urls. Only broken urls should be reported. I'll fix this soon.

**Comment by michaeljhopkins** [comment]
rockin, thanks dude.

**Comment by freekmurze** [comment]
This is fixed in v1.0.1
